### ISSUE TITLE
Outdated

### ISSUE THREAD

**Issue by fgsfdss** [elysium-project/database#2]
The last commit was at the 9th of March. Many values have changed since then, and these are becoming obsolete and incorrect.

Will we see an up-to-date commit soon?

**Comment by m-laniakea** [comment]
Next couple of days

**Comment by Locario** [comment]
Will be esES translations merged aswell?

https://github.com/Mrbytes/EsWoW2_MaNGOSR2/tree/master/mangosR2

**Comment by m-laniakea** [comment]
@Locario Don't we have Spanish translations for most things?

**Comment by Locario** [comment]
no, actually there are lots of npc texts and big other stuff without translate, only need to see the tables, my link can provide most of the esES locale translated

**Comment by omakaroni** [comment]
How do i find the armor values on bosses?

**Comment by m-laniakea** [comment]
@Locario I'll take care of the npc texts, but what other big stuff do you mean?
@omakaroni Go into creature_template, then search by the creature entry or name.

**Comment by Locario** [comment]
creature_ai_texts
db_script_strings
locales_goosip_menu_option
npc_text
locales_pagetext
locales_points of interest
mangos_string

all that tables are not translated, if you use my link all this tables can get populated with a total esES traslate

**Comment by m-laniakea** [comment]
@Locario Right, but you're assuming all the entry numbers are the same. Importing db_script_strings correctly, for example. Someone else will have to prepare that.

**Comment by Locario** [comment]
the entry id and texts are correct

**Comment by Locario** [comment]
Sql files prepared to inyect in database, can be merged with the pull request plz? tnx

https://mega.nz/#!6IRn0Q5B!6eOirVLBIkI1NgvM80WYhsjanU1NKTtIaeT-Jy7Giq0

**Comment by Locario** [comment]
https://mega.nz/#!WBZSgJza

This rar file content are sql files with "update" statement to update actual db to esES locales translated
